src/thermostat.js

```diff
@@ -173,7 +173,6 @@ export default function createThermostat({ Service, Characteristic }) {
         fn(...args)
         this.startDisconnectTimeout()
       }, (err) => {
-        err.isError = true
```

Comment:
but in `discover` method there is error catching block and it actually rejects promise - do we need to catch it additionally ? :)